### ISSUE TITLE
The content of the article is inconsistent with the code

### ISSUE THREAD

**Issue by Flyingdog-Huang** [Visual-Attention-Network/SegNeXt#46]
In codes,
        inputs = torch.cat(inputs, dim=1)
https://github.com/Visual-Attention-Network/SegNeXt/blob/c87bcaef22ed2ad72ed3decb3727e5f879343516/mmseg/models/decode_heads/ham_head.py#L233

In paper,
 our decoder only receives features from the last three stages

can you tell me what happened

**Comment by 3maoyap** [comment]
You should learn the parameter "in_index".
